tests/unit/test_pretty.py

```diff
@@ -37,6 +37,7 @@ def test_medium():
     version='3.6.1',
     region='localhost',
     model_status=StatusInfo(current='available', since='24 Feb 2025 12:02:57+13:00'),
+    sla='unsupported',
```

Comment:
This is present in 2.9 but not in 3.0. For the purposes of checking the pretty functionality, it seems simplest (in this branch) to just always have a value.